astropy/nddata/ccddata.py

```diff
@@ -578,6 +599,10 @@ def fits_ccddata_reader(filename, hdu=0, unit=None, hdu_uncertainty='UNCERT',
 
         .. versionadded:: 3.1
 
+    hdu_psf : str or None, optional
+        FITS extension from which the psf image should be initialized. If the
+        extension does not exist the psf of the CCDData is ``None``.
```

Comment:
```suggestion
        extension does not exist the psf of the CCDData is `None`.
```

Reply:
Sorry about that, I thought I had applied them all and then put the other changes on top of it.